microsoft-edge/devtools-guide-chromium/accessibility/reference.md

```diff
@@ -5,11 +5,22 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 06/07/2021
+ms.date: 12/01/2023
 ---
 # Accessibility-testing features
 
-To test your webpages for accessibility, first make a checklist of accessibility aspects to test, and then use the relevant DevTools features to check each aspect.
+This article lists the accessibility aspects to test, and the corresponding features in Microsoft DevTools.
+
+Testing the accessibility of your website is important to ensure that people with different needs can use your website. In the best scenario, you'd have access to testers with different accessibility needs, and testers using various environments. But you can already do a lot by using automated tools.
+
+For example, the [Issues tool](../issues/index.md), [Accessibility Insights](https://accessibilityinsights.io), and [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/) are tools that automatically generate accessibility reports for a webpage.  Getting an automated report from such tools is only the beginning of your accessibility-testing journey.
+
+Accessibility testing can't be fully automated and needs verification by a human navigating the product. You can do a lot yourself, for example:
```

Comment:
i'm lost in the new intro section because its argument is in the form of:
x; however, y; however, z; however, w

too many negatives, can we shift to more positive framing in the intro?